h-util-ui/Electron/operations/modules/movCompress.handler.ts

```diff
@@ -31,7 +31,7 @@ const movCompressHandler: ModuleHandler = {
         const sizeAfter = await getFileSize(filePath, 'number');
         const reduced = sizeBefore - sizeAfter;
 
-        void addNumericalStat('bytesShaved', reduced);
+        if (opts.context?.pipelineId) void addPipelineRunStat(opts.context.pipelineId, 'bytes_compressed', reduced);
```

Comment:
Todo: this should not be conditional any more, context should always be provided for common shared data.